File: 2441-largest-positive-integer-that-exists-with-its-negative/2441-largest-positive-integer-that-exists-with-its-negative.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxK = function(nums) {
    let out=0
    for(let i=0;i<nums.length;i++){
    let val=Math.max(...nums)
    if(nums.includes(-val)){
      out =val
    }else{
        nums.splice(nums.indexOf(val),1)
        i--
    }
    }
return out==undefined || out==0 ? -1:out

};

File: 2441-largest-positive-integer-that-exists-with-its-negative/2441-largest-positive-integer-that-exists-with-its-negative.py
class Solution(object):
    def findMaxK(self, nums):
        out = 0
        while nums:
            val = max(nums)
            if -val in nums:
                out = val
            else:
                nums.remove(val)
            if out != 0:
                break
        return -1 if out == 0 else out
        